receiver/mysqlreceiver/metadata.yaml

```diff
@@ -279,3 +289,21 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.perf.events.statements:
+    enabled: true
+    description: Summary of current and recent statement events.
+    unit: 1
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes: [schema, digest, digest_text, event_states]
+  mysql.perf.events.statements.wait.time:
+    enabled: true
+    description: The total wait time of the summarized timed events.
+    unit: ms
```

Comment:
should be `ps` according to mysql documentation: https://dev.mysql.com/doc/refman/5.7/en/performance-schema-events-statements-current-table.html

Reply:
Some other metrics added to this receiver recently were also in picoseconds, but we decided to convert to nanoseconds. Context: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14328#issuecomment-1255060567

I'm not sure that was strictly necessary, but we should probably align these one way or the other.

Reply:
Nanoseconds sounds reasonably for now. Will change that